tool/build_release.sh

```diff
@@ -10,15 +10,41 @@ pushd packages/devtools_app
 
 rm -rf build
 rm -rf ../devtools/build
+
+# This avoids requiring an internet connection for CanvasKit at runtime.
+# This URL should be updated to keep in sync with the version from the engine.
+# See https://github.com/flutter/engine/blob/353efcdf3c0ec0ecf0275d55e4a22329397f899f/lib/web_ui/lib/src/engine/canvaskit/initialization.dart#L51-L79,
+# but compare with the code in master for getting the current version.
```

Comment:
We need a way to check that automatically or we will soon have bugs due to using the wrong canvaskit version.
I'd be fine with something as crude as grepping that https://github.com/flutter/engine/lib/web_ui/lib/src/engine/canvaskit/initialization.dart 
contains exactly https://unpkg.com/canvaskit-wasm@0.22.0/bin/
and failing if it doesn't.
It might also make sense to rewrite this release script in Dart now that it is getting more complex.

Reply:
I'm agree jacob that we need to include an automated check in tandem with this change.

Reply:
It'd probably be good if the web engine just included a manifest somewhere of what files/URLs it does/might access at runtime. Checking with some web folks about that.

Reply:
Added a grep for it in the script. This is .. pretty hacky. It'd be a lot better to have a manifest of some sort. But I tried with an invalid canvaskit URL and it works. It will break if the initialization code gets moved around, or Flutter changes internal cache stuff.

I think it's probably ok until some better option is available, since this is just a build script run by developers who hopefully know what's going on.

Reply:
Can you file a bug to move to a better solution and link to it in comments here?

Reply:
Added a couple bugs.